app/retail/emails.py

```diff
@@ -521,6 +521,40 @@ def render_start_work_applicant_expired(interest, bounty):
     return response_html, response_txt, subject
 
 
+def render_notify_funder(profile):
+    from dashboard.models import Bounty, NotifyFunder
+    from django.db.models import F, Func
+    from collections import Counter
```

Comment:
😁 Aye, can you move these imports to the top of the file?

Reply:
will do ^_^